app/src/main/res/values/strings.xml

```diff
@@ -191,4 +191,6 @@
   <string name="your_languages">Selected languages:</string>
   <string name="other_languages">Other languages:</string>
   <string name="no_items_msg">No items available</string>
+  <string name="sharing_message">The app is a lightweight piece of software reading bigger files stored on your device or SD card: once it is installed, you can select which additional content you would like to download (Wikipedia, Wiktionary, TED talks, etc.) and be ready for when your internet connection is bad. Download Kiwix Now:https://play.google.com/store/apps/details?id=org.kiwix.kiwixmobile </string>
```

Comment:
Maybe use our Google Play description instead.

Reply:
It is already from google play's description.

Reply:
Cool, we should update this at some point.